src/Core/Form/IdentifiableObject/DataProvider/PartialRefundFormDataProvider.php

```diff
@@ -54,6 +54,7 @@ public function __construct(
      */
     public function getData($orderId)
     {
+        /** @var OrderForViewing $orderForViewing */ 
```

Comment:
It was a suggestion, but the `use` statement is missing I think

Reply:
Well I don't need the use statement for OrderForViewing, I need the one for `GetOrderForViewing`.